### PR TITLE
Server cleanup

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

**Review by jakemac53 — APPROVED** [review]

File: dart/tools/dev_server/bin/server.dart [hunk 1]
```diff
@@ -3,12 +3,15 @@ import 'dart:convert';
 import 'dart:io';
 
 import 'package:args/args.dart';
-import 'package:path/path.dart' as path;
+import 'package:path/path.dart' as p;
```
Comment (natebosch) [inline dart/tools/dev_server/bin/server.dart]: why does this need to be shortened?
`as path` is (very slightly) more common, and IMO easier to read at usage sites

File: dart/tools/dev_server/bin/server.dart [hunk 2]
```diff
@@ -3,12 +3,15 @@ import 'dart:convert';
...
 import 'package:shelf/shelf.dart';
 import 'package:shelf/shelf_io.dart' as io;
 import 'package:shelf_static/shelf_static.dart';
 import 'package:watcher/watcher.dart';
 
+String get _workspacePath => p.join(Platform.environment['RUNFILES'],
```
Comment (natebosch) [inline dart/tools/dev_server/bin/server.dart]: do we expect this to change at runtime? Should it be a final variable instead?

File: dart/tools/dev_server/bin/server.dart [hunk 3]
```diff
@@ -18,12 +21,14 @@ Future main(List<String> args) async {
   // --build-target and --watch have to be used together
   if (_buildTarget.isEmpty != watchPaths.isEmpty) {
     print('--build-target and --watch arguments have to be used together');
-    exit(1);
+    // command line usage error
+    exitCode = 64;
```
Comment (natebosch) [inline dart/tools/dev_server/bin/server.dart]: 👍

File: dart/tools/dev_server/bin/server.dart (hunk 1 above)
Comment (kevmoo) [inline dart/tools/dev_server/bin/server.dart]: Tis tradition – `path` is a **VERY** common variable name.

File: dart/tools/dev_server/bin/server.dart (hunk 2 above)
Comment (kevmoo) [inline dart/tools/dev_server/bin/server.dart]: true true

**Pull request merged**